test/zdtm.py

```diff
@@ -256,11 +266,18 @@ def __construct_root(self):
         for ldir in ["/bin", "/sbin", "/lib", "/lib64"]:
             os.symlink(".." + ldir, self.root + "/usr" + ldir)
 
+    def __construct_dev(self):
+        for dir in self.__dev_dirs:
+            os.mkdir(os.path.join(self.devpath, dir))
+            os.chmod(os.path.join(self.devpath, dir), 0o777)
```

Comment:
Could we use `os.mkdir(os.path.join(self.devpath, dir), mode=0o777)` here?
https://docs.python.org/3/library/os.html#os.mkdir

Reply:
In this script, we use chmod for all new directories. I think the problem here is that umask is masked out from mode specified for mkdir.